### PR TITLE
[DASH] Add DASH Label tag in mpd.

### PR DESCRIPTION
**Pull request by sr1990**
This PR is related to https://github.com/google/shaka-packager/issues/881

```
../packager \                                                                   
  'in=Sintel-1024x576.mp4,stream=video,init_segment=h264_1024p/init.mp4,segment_template=h264_1024p/$Number$.m4s,dash_label=Hindi' \
  'in=Sintel-1280x720.mp4,stream=video,init_segment=h264_1280p/init.mp4,segment_template=h264_1280p/$Number$.m4s,dash_label=English' \
  'in=Sintel-audio.mp4,stream=audio,init_segment=audioSintel/init.mp4,segment_template=audioSintel/$Number$.m4s' \
  --mpd_output h264_label.mpd --generate_static_live_mpd
```
will create the following mpd

```
<!--
Generated with https://github.com/google/shaka-packager version b7ef11fa70-debug
-->
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:mpeg:dash:schema:mpd:2011 DASH-MPD.xsd" profiles="urn:mpeg:dash:profile:isoff-live:2011" minBufferTime="PT2S" type="static" mediaPresentationDuration="PT52.20833206176758S">
<Period id="0">
<AdaptationSet id="0" contentType="video" width="1280" height="720" frameRate="96/4" segmentAlignment="true" par="16:9" Label="English">...</AdaptationSet>
<AdaptationSet id="1" contentType="audio" segmentAlignment="true">...</AdaptationSet>
<AdaptationSet id="2" contentType="video" width="1024" height="576" frameRate="96/192" segmentAlignment="true" par="16:9" Label="Hindi">...</AdaptationSet>
</Period>
</MPD>
```

### CODE REVIEW

**Comment by sr1990** [comment]
Label is an element. Will update the PR.

**Comment by sr1990** [comment]
Label added as an element
```
<!--...-->
<MPD xmlns="urn:mpeg:dash:schema:mpd:2011" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:mpeg:dash:schema:mpd:2011 DASH-MPD.xsd" profiles="urn:mpeg:dash:profile:isoff-live:2011" minBufferTime="PT2S" type="static" mediaPresentationDuration="PT52.20833206176758S">
<Period id="0">
<AdaptationSet id="0" contentType="video" width="1024" height="576" frameRate="96/192" segmentAlignment="true" par="16:9">
<Label>Hindi</Label>
<Representation id="0" bandwidth="212490" codecs="avc1.4d401f" mimeType="video/mp4" sar="1:1">...</Representation>
</AdaptationSet>
<AdaptationSet id="1" contentType="audio" segmentAlignment="true">...</AdaptationSet>
<AdaptationSet id="2" contentType="video" width="1280" height="720" frameRate="96/4" segmentAlignment="true" par="16:9">
<Label>English</Label>
<Representation id="2" bandwidth="2942370" codecs="avc1.4d401f" mimeType="video/mp4" sar="1:1">...</Representation>
</AdaptationSet>
</Period>
</MPD>
```

File: packager/mpd/base/mpd_utils.cc [hunk 1]
```diff
@@ -158,6 +158,9 @@ std::string GetAdaptationSetKey(const MediaInfo& media_info,
     key.append("unknown:");
   }
 
+  if (media_info.has_dash_label())
+    key.append(media_info.dash_label() + ":");
+
```
Comment (sr1990) [inline packager/mpd/base/mpd_utils.cc:163]: Taking Label into consideration while creating AdaptationSet key.

**Comment by joeyparrish** [comment]
There is a minor conflict in the docs, but I should be able to review it anyway.  This is next in my list of PRs to review.  Thanks for your patience as we work to revive this project!

**Comment by hariszukanovic** [comment]
Any chance for this merge?
Actually, there seems to be no other way to control the labels for audio tracks

**Comment by sr1990** [comment]
will create a new pr. Closing this.

**Pull request closed without merging**